utils/formats/sarifutils/test_sarifutils.go

```diff
@@ -1,6 +1,8 @@
 package sarifutils
```

Comment:
It is a bit confusing that we have sarifutils_test.go and test_sarifutils.go
This file contains functions to be used in many tests files.
Can we call it sarifutils_common.go or something like that?

Reply:
it is not.
files with X_test.go will not be included in the library.
files with test_x.go have utils for tests that can be used (funcs that uses t.Testing)